src/Microsoft.ML.Legacy/Runtime/Internal/Tools/CSharpGeneratorUtils.cs

```diff
@@ -296,7 +296,7 @@ private static string GetCharAsString(char value)
                     if (generatedClasses.IsGenerated(fieldType.FullName))
                         return generatedClasses.GetApiName(fieldType, rootNameSpace) + "." + enumAsString;
                     else
-                        return generatedClasses.GetApiName(fieldType, "Runtime") + "." + enumAsString;
+                        return generatedClasses.GetApiName(fieldType, "") + "." + enumAsString;
```

Comment:
fyi -- so anyone else reading this doesn't go and check, `GetApiName()` seems to handle `""`:
https://github.com/dotnet/machinelearning/blob/a8b844ccee21b7639e1b0c24fefd1e69088f0d7e/src/Microsoft.ML.Legacy/Runtime/Internal/Tools/GeneratedClasses.cs#L27-L38